rlclientlib/live_model_impl.cc

```diff
@@ -243,6 +247,36 @@ namespace reinforcement_learning {
     return error_code::success;
   }
 
+  int live_model_impl::request_multi_slot_decision(const char * context_json, unsigned int flags, multi_slot_response_detailed& resp, api_status* status)
+  {
+	  const auto uuid = boost::uuids::to_string(boost::uuids::random_generator()());
+	  return request_multi_slot_decision(uuid.c_str(), context_json, flags, resp, status);
+  }
+
+  int live_model_impl::request_multi_slot_decision(const char * event_id, const char * context_json, unsigned int flags, multi_slot_response_detailed& resp, api_status* status)
+  {
+    resp.clear();
+
+    std::vector<std::string> slot_ids;
+    std::vector<std::vector<uint32_t>> action_ids;
+    std::vector<std::vector<float>> action_pdfs;
+    std::string model_version;
+
+    RETURN_IF_FAIL(live_model_impl::request_multi_slot_decision_shared(event_id, context_json, slot_ids, action_ids, action_pdfs, model_version, status));
```

Comment:
I suggest to rename it to maybe request_multi_slot_decision_impl instead of request_multi_slot_decision_shared.
Reasons:
- there is nothing inside this function about sharing something with someone, and we do not want to rename it if we suddenly decide to remove one of request_multi_slot_decision calls (and it stopped being shared actually)
- in this context it is a bit ambiguous with "shared" features (it has nothing to do with that, but can give that impression)